### ISSUE TITLE
[Accessibility] List of buttons with missing role

### ISSUE THREAD

**Issue by michalina-graczyk** [mirumee/saleor-storefront#829]
### Steps to reproduce the problem
1. Test the main page for a accessibilities

**System information**
Operating system: Mac OS
Browser: Chrome v. 84

Rule | Selector | Guide name | Image
-- | -- | -- | --
button-missing-role | ["body > div#root:nth-of-type(1) > header:nth-of-type(1) > nav#header.main-menu:nth-of-type(1) > div.main-menu__right:nth-of-type(3) > ul:nth-of-type(1) > div.menu-dropdown:nth-of-type(1) > li.main-menu__icon.main-menu__user--active:nth-of-type(1) > div:nth-of-type(1) > div:nth-of-type(1) > svg.injected-svg:nth-of-type(1)"] | Buttons and links | ![image](https://user-images.githubusercontent.com/25908728/89187612-d0ed0f00-d59d-11ea-9db6-c0a39d5bf052.png)
button-missing-role | ["body > div#root:nth-of-type(1) > header:nth-of-type(1) > nav#header.main-menu:nth-of-type(1) > div.main-menu__right:nth-of-type(3) > ul:nth-of-type(1) > li.main-menu__icon.main-menu__cart:nth-of-type(1)"] | Buttons and links | ![image](https://user-images.githubusercontent.com/25908728/89187803-0abe1580-d59e-11ea-9144-a104127720af.png)
